docs/manual/faq/_index.de.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Fragen und Antworten"
+description: "Antworten zu den häufigsten Fragen."
+url: "faq"
+aliases:
+    - /de/faq/
+weight: 110
+---
+
+Hier findest du eine Sammlung der häufigsten Fragen mit entsprechenden Lösungen. 
+
+Wenn du selbst einen Vorschlag für diesen Bereich hast, verwende den Link »Diese Seite bearbeiten« oben rechts. 
+Besitzt du ein GitHub-Konto und bist angemeldet erstellt GitHub automatisch einen Fork in dem du 
+deine Vorschläge hinzufügen kannst. Anschließend kannst du über GitHub einen Pull-Request erstellen.
+</br></br>
+
+{{% expand "Ich habe mein Administrator-Passwort vergessen." %}}
+Entferne hierzu in der Tabelle »tl_user« deiner Datenbank in der Spalte »admin« den Wert »1«. Anschließend kannst
```

Comment:
BTW: Falls es in der Tabelle »tl_user« mehrere Datensätze gibt, bei denen das Admin-Flag gesetzt ist, müsste man den Wert zunächst bei allen zurücksetzen, um anschließend im Contao Installtool einen neuen Administrator anlegen zu können.